index.js

```diff
@@ -8,15 +8,20 @@ function createRavenMiddleware(Raven, options = {}) {
     actionTransformer = identity,
     stateTransformer = identity,
     breadcrumbCategory = "redux-action",
-    filterBreadcrumbActions = filter
+    filterBreadcrumbActions = filter,
+    userContextStateTransformer
```

Comment:
Maybe we could call it `getUserContext`? That would fit with the convention i’ve seen for naming selectors which, as you show in your example, this is.

Reply:
Agree! That is a much better name!